playbooks/roles/suse-build-images/defaults/main.yml

```diff
@@ -21,3 +21,6 @@ build_infra_components_list:
 suse_build_images_packages:
   - docker
   - gnu_parallel
+
+sle_utils_repository_url:
+  - "https://download.opensuse.org/repositories/utilities/SLE_12"
```

Comment:
Please put this on the same line (and remove the  - ) as the "sle_utils_repository_url" to be a string.